1920.build-array-from-permutation.js

```diff
@@ -0,0 +1,18 @@
+/*
+ * @param {number[]} nums
+ * @return {number[]}
+*/
+
+/* URL of this problem
+    https://leetcode.com/problems/build-array-from-permutation/
+*/
+
+ var buildArray = function(nums) {
+    let ans = [];
+    
+    for (let i = 0; i < nums.length; i++) {
+        ans.push(nums[nums[i]]);
+    }
+    
+    return ans
```

Comment:
can you use array.prototype.map to solve this problem?

Reply:
I solved this with array.prototype.map and modified the code.